_aicspylibczi/ImageFactory.cpp

```diff
@@ -104,34 +128,37 @@ namespace pylibczi {
       shape.emplace_back(size.h);
       shape.emplace_back(size.w);
 
-      std::shared_ptr<Image> image = s_pixelToImage[pixelType](shape, pixelType, plane_coordinate_, box_, index_m_);
+      size_t mem_index = 0;
+
+      std::shared_ptr<Image> image = s_pixelToImage[pixelType](shape, pixelType, plane_coordinate_, box_,
```

Comment:
Something feels weird about how s_pixelToImage is named.   
Maybe renaming it to something like s_pixelTypeToImageConstructor, and/or :
```
auto imageFactoryFunction = s_pixelToImage[pixelType];
std::shared_ptr<Image> image = imageFactoryFunction(...)
```

Reply:
or even something like s_getImageFactoryFunction, just more descriptive of its usage.